joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/AstForStructuresCreator.scala

```diff
@@ -24,7 +24,7 @@ trait AstForStructuresCreator(implicit withSchemaValidation: ValidationMode) { t
 
```

Comment:
Rename this file to `AstForTypeDeclCreator` as that is more readable

Reply:
Addressed here: https://github.com/joernio/joern/pull/3585/commits/6461283b1309525a09028e9c7812f0602c880e34